info.mumble.Mumble.yml

```diff
@@ -0,0 +1,97 @@
+id: info.mumble.Mumble
+rename-desktop-file: mumble.desktop
+rename-icon: mumble
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: mumble
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-2
+separate-locales: false
+cleanup:
+  - /usr/include
```

Comment:
It should be just `/include`, flatpak-builder automatically adds `/app` prefix. Please add `/lib/pkgconfig` too.

Reply:
Looking at other files, does mumble need all these binaries?

```
[📦 info.mumble.Mumble app]$ ls /app/bin/
avahi-browse		   avahi-publish-address  avahi-resolve-host-name  bvnc
avahi-browse-domains	   avahi-publish-service  avahi-set-host-name	   mumble
avahi-discover-standalone  avahi-resolve	  bshell		   mumble-overlay
avahi-publish		   avahi-resolve-address  bssh			   protoc
```